_data/changelogs/component-icon.yml

```
@@ -2,6 +2,11 @@ title: Icon
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Removed deprecated xlink instances.
```

Since this page will also get a changelog for the uswds change in PR #2951, I think we should make a summary that is more specific to the guidance update.
```suggestion
    summary: Removed deprecated `xlink` references from code samples. 
```

---

Resolved in [184feb7](https://github.com/uswds/uswds-site/pull/2926/commits/184feb776932fed5edf4ea99d10321a9161fe77c)